src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -27,6 +27,11 @@ import { STORAGE_KEY } from '../constants/storageKeys';
 const DEFAULT_CTR_RADIUS_KM = 80;
 const DEFAULT_CTR_CEILING_FT = 10000;
 const DEFAULT_INITIAL_ALTITUDE_FT = 5000;
+const DEFAULT_RANGE_RINGS = {
+    enabled: false,
```

Comment:
if this is a constant, the child keys should be `UPPERCASE` as well

Reply:
It's actually a default object which is directly assigned to a property during initialization, so the keys have to match the casing of the object. Is this a bad practice, or okay to do it like this?

Reply:
Now that you mention it, I think we should create a copy of the object during initialization, so we don't (accidentally) modify the constant.